apps/prairielearn/python/prairielearn/conversion_utils.py

```diff
@@ -94,6 +94,13 @@ class _JSONSerializedSympyMatrix(_JSONSerializedGeneric):
 
 
 def is_int_json_serializable(n: int) -> bool:
+    """Check if an integer is less than Number.MAX_SAFE_INTEGER and greater than Number.MIN_SAFE_INTEGER.
+
+    See <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER>.
+
+    Returns:
+        True if it can be serialized by JS code.
```

Comment:
```suggestion
        `True` if it can be serialized by JS code.
```

Reply:
Resolved in [163dfab](https://github.com/PrairieLearn/PrairieLearn/pull/11758/commits/163dfab708fb88fca0820d6e6f061f20e2cddebe).